### ISSUE TITLE
Regular connect-sync crash

### ISSUE THREAD

**Issue by jamesloosli** [1Password/onepassword-operator#115]
### Your environment

<!-- Version of the Operator when the error occurred -->
Operator Version: 1.4.1

<!-- What version of the Connect server are you running?
You can get this information from the Integrations section in 1Password
https://start.1password.com/integrations/active
-->
Connect Server Version: 1.5.1

<!-- What version of Kubernetes have you deployed the operator to? -->
Kubernetes Version: 1.22

## What happened?
<!-- Describe the bug or error -->
Without warning, the onepassword connect-sync pod will fail to start.

## What did you expect to happen?
<!-- Describe what should have happened -->
I'd like the pod to run without crashing?

## Steps to reproduce
1. Install onepassword-connect on a kubernetes cluster, with a valid token.
2. Wait
3. ???

## Notes & Logs
<!-- Paste any logs here that may help with debugging.
Remember to remove any sensitive information before sharing! -->
```
{"log_message":"(I) disabling bus peer auto-discovery","timestamp":"2022-05-26T22:55:17.751628353Z","level":3}
{"log_message":"(I) connected to bus peer at localhost:11220","timestamp":"2022-05-26T22:55:17.75261625Z","level":3}                                                                                                                            
{"log_message":"(W) configured to use HTTP with no TLS","timestamp":"2022-05-26T22:55:17.752676703Z","level":2}
{"log_message":"(I) starting 1Password Connect Sync ...","timestamp":"2022-05-26T22:55:17.752867812Z","level":3}                                                                                                                                
{"log_message":"(I) serving on :8081","timestamp":"2022-05-26T22:55:17.752917666Z","level":3}
{"log_message":"(I) database initialization complete","timestamp":"2022-05-26T22:55:17.753478884Z","level":3}                                                                                                                                   
Error: checking integrity of file : reading file: read /home/opuser/.op/data/files: is a directory
Usage:                                                                                                                                                                                                                                            
connect-sync [flags]
                                                                                                                                                                                                                                                
Flags:
  -h, --help      help for connect-sync                                                                                                                                                                                                           
-v, --version   version for connect-sync
```

**Comment by wise0wl** [comment]
This has been a serious issue for some time that we have to constantly work around.

**Comment by jillianwilson** [comment]
Hi there!

Sorry to hear you are running into this frustrating issue. Let me clarify things first so I can help debug; are you able to get connect working at all, ie is it failing at every startup, during some startups, or can you get the sync to run for some time before crashing?

**Comment by jpcoenen** [comment]
Hey @wise0wl,

I have been able to reproduce a problem that looks very similar to what you are describing. We're working on a fix. I'll report back when the fix has been released.

**Comment by jpcoenen** [comment]
We have released v1.5.4 of Connect which contains a fix for a similar problem. Could you let me know if updating resolves the issue?
